logrotate/Options.cmake

```diff
@@ -20,10 +20,10 @@
 
 
 SET(G3_LIBRARY_PATH "" CACHE FILEPATH "Path to g3log libraries")
-IF(EXISTS "${G3_LIBRARY_PATH}/libg3logger_shared.so" )
+IF(EXISTS "${G3_LIBRARY_PATH}/libg3logger.a")
```

Comment:
If you keep the library name in a variable then this would just look for the library., and be ignorant to wherever you had .a or .so suffix